assets/src/scripts/monitoring-location/components/hydrograph/index.js

```diff
@@ -20,8 +20,8 @@ import {Actions as floodDataActions} from 'ml/store/flood-inundation';
 
 import {showDataLoadingIndicator} from './data-loading-indicator';
 import {drawDataTables} from './data-table';
+import {renderDownloadLinks} from './download-links';
```

Comment:
Can we change the name to drawDownloadLinks to match the other functions?

Reply:
Sure can!